documentation/src/docs/asciidoc/release-notes-5.0.0-M4.adoc

```diff
@@ -104,7 +104,9 @@ on GitHub.
 * `TestInstancePostProcessors` registered on test methods are now invoked.
 * There are two new signatures for `Assertions.fail`: `Assertions.fail(Throwable cause)` and
   `Assertions.fail(String message, Throwable cause)`.
-
+* New `Assertions.assertLinesMatch()` comparing lists of `String` featuring `Object::equals` and
+  regular expression checks. It also provides a fast-forward mechanism to skip actual lines that
+  are expected to change in each invocation, e.g. duration, timestamps and stack traces.
```

Comment:
Mh, that seems a bit too detailed for a short note. I'll add the _See the Javadoc for more details._ with a link to it, though.